src/Select/Select.js

```diff
@@ -0,0 +1,218 @@
+  border-radius: ${({ theme }) => theme.borderRadiusNormal};
+  font-family: ${({ theme }) => theme.fontFamily};
+  font-size: ${({ theme, size }) =>
+    size === SIZE_OPTIONS.SMALL ? theme.fontSizeInputSmall : theme.fontSizeInputNormal};
+  color: ${({ theme, isPlaceholder }) =>
+    isPlaceholder ? theme.paletteInkLight : theme.colorTextInput};
+  background: ${({ theme }) => theme.backgroundInput};
+  padding: ${({ theme, size }) =>
+    size === SIZE_OPTIONS.SMALL
+      ? `${theme.spaceXSmall} ${theme.spaceXLarge} ${theme.spaceXSmall} ${theme.spaceSmall}`
+      : `${theme.spaceSmall} ${theme.spaceXXLarge} ${theme.spaceSmall} ${theme.spaceMedium}`};
+  padding-left: ${({ prefix }) => prefix && "48px"}; // temporary replacement for token
+  appearance: none;
+  outline: none;
+  cursor: pointer;
+  /* for IE10 */
+  &::-ms-expand {
+    display: none;
+  }
+  &:disabled {
+    color: ${({ theme }) => theme.colorTextInputDisabled};
+    background: ${({ theme }) => theme.backgroundInputDisabled};
+    cursor: default;
+  }
+`;
+
+const SelectSuffix = styled(({ theme, disabled, ...props }) => <div {...props} />)`
+  position: absolute;
+  right: ${({ theme }) => theme.spaceXSmall};
+  color: ${({ theme, disabled }) =>
+    !disabled ? theme.colorTextInput : theme.colorTextInputDisabled};
+  pointer-events: none;
+  & > * {
+    width: ${({ theme, size }) => size === SIZE_OPTIONS.SMALL && theme.widthIconSmall};
+    height: ${({ theme, size }) => size === SIZE_OPTIONS.SMALL && theme.heightIconSmall};
+    margin-bottom: ${({ size }) =>
+      size === SIZE_OPTIONS.SMALL && "2px"}; // temporary replacement for token spaceXXXSmall
+  }
+`;
+
+class Select extends React.Component<Props, State> {
+  static defaultProps = {
+    value: "",
+    size: SIZE_OPTIONS.NORMAL,
+    disabled: false,
+    theme: defaultTokens,
+  };
+
+  state = {
+    value: this.props.value,
+    active: false,
+  };
+
+  handleChange = (e: SyntheticInputEvent<HTMLSelectElement>) => {
+    this.setState({ value: e.target.value });
+    this.props.onChange(e);
+  };
+
+  handleFocus = (e: SyntheticInputEvent<HTMLSelectElement>) => {
+    const { onFocus } = this.props;
+
+    this.setState({ active: true });
+    if (onFocus) {
+      onFocus(e);
+    }
+  };
+
+  handleBlur = (e: SyntheticInputEvent<HTMLSelectElement>) => {
+    const { onBlur } = this.props;
+
+    this.setState({ active: false });
+    if (onBlur) {
+      onBlur(e);
+    }
+  };
+
+  renderOption = ({ value, label, disabled }: Option) => (
+    <option key={`option-${value}`} value={value} label={label || value} disabled={disabled}>
+      {label || value}
+    </option>
+  );
+
+  render() {
+    const { theme, prefix, label, size, disabled, placeholder, help, error, options } = this.props;
+    const { value, active } = this.state;
+    const isPlaceholder = placeholder && value === "";
+
+    return (
+      <React.Fragment>
+        {label && (
+          <SelectLabel theme={theme} value={value}>
```

Comment:
Should be `select` focused and opened after clicking on `label`?

Reply:
Can make this when we will have custom select ready. For now, it is not so much important.